sunpy/visualization/animator/wcs.py

```diff
@@ -80,13 +80,6 @@ def __init__(self, data, wcs, slices, coord_params=None, ylim='dynamic', ylabel=
             image_axes.append(slices[::-1].index("y"))
             self.plot_dimensionality = 2
 
-        if self.plot_dimensionality == 1:
-            try:
-                from astropy.visualization.wcsaxes.frame import RectangularFrame1D  # NOQA
```

Comment:
Do we still not need this import for something?

Reply:
In `wcs.py` it is the only mention of it

Reply:
Must have just been a check then.

Reply:
yeah this was how I was checking if you were running an astropy version which supported 1D or not.